File: 7-kyu/pair-zeros/javascript/solution.js
function pairZeros(arr) {
  let arry = arr;
  for (let i = 0; i < arry.length; i++) {
    if (arry[i] === 0) {
      for (let j = i + 1; j < arry.length; j++) {
        if (arry[j] === 0) {
          arry[j] = -1;
          break;
        }
      }
    }
  }
  const paired = arry.filter(elem => elem !== -1);
  return paired;
}

File: 7-kyu/pair-zeros/python/solution.py
def pair_zeros(arr):
    arry = []
    cnt = 0
    for n in arr:
        if n == 0:
            cnt += 1
            if cnt % 2 == 1:
                arry.append(n)
        else:
            arry.append(n)
    return arry
